hack/generator/pkg/astmodel/allof_type.go

```diff
@@ -159,3 +159,19 @@ func (allOf *AllOfType) String() string {
 
 	return fmt.Sprintf("(allOf: %s)", strings.Join(subStrings, ", "))
 }
+
+// DebugDescription adds a description of the current AnyOf type to the passed builder
```

Comment:
minor: Should be `WriteDebugDescription`?

Reply:
Fixed.